src/ray/raylet/agent_manager.cc

```diff
@@ -56,58 +56,65 @@ void AgentManager::StartAgent() {
     return;
   }
 
-  if (RAY_LOG_ENABLED(DEBUG)) {
-    std::stringstream stream;
-    stream << "Starting agent process with command:";
-    for (const auto &arg : options_.agent_commands) {
-      stream << " " << arg;
-    }
-    RAY_LOG(DEBUG) << stream.str();
-  }
-
-  // Launch the process to create the agent.
-  std::error_code ec;
+  // Create a random agent_id to pass to the child process
```

Comment:
```suggestion
  // Create a random agent_id to pass to the child process. We cannot use pid an id because os.getpid() from the python process is not reliable in Windows. See [Github issue link]
```